tracer/src/Datadog.Trace/Debugger/Symbols/SymbolsUploader.cs

```diff
@@ -70,7 +70,9 @@ internal class SymbolsUploader : IDebuggerUploader
             _enablementSemaphore = new SemaphoreSlim(0);
             _thresholdInBytes = settings.SymbolDatabaseBatchSizeInBytes;
             _symDb3rdPartyIncludes = settings.SymDbThirdPartyDetectionIncludes;
+            _symDb3rdPartyIncludes.UnionWith(settings.ThirdPartyDetectionIncludes);
             _symDb3rdPartyExcludes = settings.SymDbThirdPartyDetectionExcludes;
+            _symDb3rdPartyExcludes.UnionWith(settings.ThirdPartyDetectionExcludes);
```

Comment:
This _modifies_ the hashset stored in `settings.SymDbThirdPartyDetectionIncludes` (because you only copy the reference into `_symDb3rdPartyIncludes`). I suspect what you really want to do is create a _new_ hashset that is the union? You can do this nicely with spread collections now 🙂 

```suggestion
            _symDb3rdPartyIncludes = new HashSet<string>([..settings.SymDbThirdPartyDetectionIncludes, ..settings.ThirdPartyDetectionIncludes]);
            _symDb3rdPartyExcludes = new HashSet<string>([..settings.SymDbThirdPartyDetectionExcludes, ..settings.ThirdPartyDetectionExcludes]);
```

Reply:
The intention was to copy a reference (and modify `settings.SymDbThirdPartyDetectionIncludes`) because we do not support updating the configuration, but the new collections features are really cool.

Reply:
Fixed in c100f7c2061d3ceabe9568cf13bc7ce1cca5267f